source/CaptureOdds.cpp

```diff
@@ -186,7 +187,7 @@ vector<double> CaptureOdds::Power(const Ship &ship, bool isDefender)
 			power.insert(power.end(), it.second, value);
 	}
 	// Use the best weapons first.
-	sort(power.begin(), power.end(), greater<double>());
+	sort(parallel::par_unseq, power.begin(), power.end(), greater<double>());
```

Comment:
another small thing, how often is this even called per frame?

Reply:
Yeah, this one does seem to be rather useless.